### PR TITLE
Jonah Park html-forms Exercise

### PR DESCRIPTION
**Pull request by jonahp001**
Kapture 2023-01-24 at 12 18 39

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    </div>
+    <div>
+      <p>Would you like to sign up for more information?</p>
+      <input id="yessignbox" type="checkbox" name="signupcheckbox" value="signedup" />
+      <label for="yessignbox">Sign me up!</label>
+    </div>
+    <div>
+      <label>Comments</label>
+      <div>
+        <textarea cols="30" rows="10"></textarea>
```
Comment (robgmerrill) [inline html-forms/index.html:41]: same feedback as line 11

**Review by robgmerrill — CHANGES_REQUESTED** [review]
Great start! Almost there.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  <form action="index.html" method="get">
+    <label for="email">Email</label>
+    <div>
+      <input id="email" type="text" name="email" maxlength="50" />
+    </div>
+    <label for="coolcoding">Why is coding cool?</label>
+    <div>
+      <select id="coolcoding" name="coolcodingoption">
+        <option value="Select an option">Select an option</option>
+        <option value="Because it's interesting">Because it's interesting</option>
```
Comment (robgmerrill) [inline html-forms/index.html:17]: small nitpick. Instead of `'` please use HTML entity `&apos;` on line 17, 18, 19

**Review by robgmerrill — CHANGES_REQUESTED** [review]
One more nitpick thing to address. Almost there!

File: html-forms/index.html (hunk 2 above)
Comment (jonahp001) [reply html-forms/index.html:17]: noted!

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**